mycity/platforms/amazon/models/en_US.json

```diff
@@ -1,97 +1,181 @@
+                    "slots": [
+                        {
+                            "name": "Address",
+                            "type": "AMAZON.PostalAddress",
+                            "samples": [
+                                "My address is {Address}",
+                                "It's {Address}",
+                                "{Address}"
+                            ]
+                        }
+                    ],
+                    "samples": [
+                        "where can I park during a snow emergency",
+                        "where can I park during snow",
+                        "where can I park",
+                        "snow parking",
+                        "emergency parking near"
+                    ]
+                },
+                {
+                    "name": "GetAlertsIntent",
+                    "slots": [],
+                    "samples": [
+                        "give me all alerts",
+                        "are there any alerts",
+                        "are there any emergencies",
+                        "are services running normally",
+                        "all alerts"
+                    ]
+                },
+                {
+                    "name": "AMAZON.HelpIntent",
+                    "samples": []
+                },
+                {
+                    "name": "UnhandledIntent",
+                    "slots": [],
+                    "samples": [
+                        "blah blah blah",
+                        "gofougildags",
+                        "heh",
+                        "testing",
+                        "this is dumb",
+                        "workaround to get around",
+                        "thinking about things",
+                        "what up doe",
+                        "dealing with it",
+                        "doing it",
+                        "whatever",
+                        "whatever and ever amen"
+                    ]
+                },
+                {
+                    "name": "AMAZON.StopIntent",
+                    "samples": []
+                },
+                {
+                    "name": "AMAZON.CancelIntent",
+                    "samples": []
+                }
+            ],
+            "types": []
         },
-        {
-          "name": "AMAZON.HelpIntent"
+        "dialog": {
+            "intents": [
+                {
+                    "name": "TrashDayIntent",
+                    "confirmationRequired": false,
+                    "prompts": {},
+                    "slots": [
+                        {
+                            "name": "Address",
+                            "type": "AMAZON.PostalAddress",
+                            "confirmationRequired": false,
+                            "elicitationRequired": true,
+                            "prompts": {
+                                "elicitation": "Elicit.Slot.604703910902.353139060048"
```

Comment:
These were all generated using the Alexa interaction model GUI, so I can't speak to exactly how they are generated. We _might_  be able to share elicitations in cases where they are the same (like the addresses here) but I am not sure it is worth it in case we want to change one in the future without affecting the other ones.